### PR TITLE
supercollider: 3.9.3 -> 3.10.2

### PR DESCRIPTION
**Pull request by MrMebelMan**
###### Motivation for this change

Updating to a newer version: https://github.com/supercollider/supercollider/releases/tag/Version-3.10.2

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/interpreters/supercollider/default.nix [hunk 1]
```diff
@@ -1,20 +1,20 @@
 { stdenv, fetchurl, cmake, pkgconfig, alsaLib
 , libjack2, libsndfile, fftw, curl, gcc
-, libXt, qtbase, qttools, qtwebkit, readline
-, useSCEL ? false, emacs
+, libXt, qtbase, qttools, qtwebkit, qtwebengine
```
Comment (veprbl) [inline pkgs/development/interpreters/supercollider/default.nix]: We probably don't need `qtwebkit` anymore. The `qtwebengine` replaces it.

Comment (MrMebelMan) [reply pkgs/development/interpreters/supercollider/default.nix]: removed it, thanks!

**Comment by veprbl** [comment]
@GrahamcOfBorg build supercollider sonic-pi supercollider_scel

**Comment by veprbl** [comment]
Changed the license as readme states:
"SuperCollider is free software available under Version 3 the GNU General Public License. See COPYING for details."
There is also this license page: http://doc.sccode.org/Other/Licensing.html
Which claims that it's GPL3+. I'm not sure if the mentioned included GPL3 code is also GPL3+.

**Review by veprbl — APPROVED** [review]

**Pull request merged**